src/rendering/RenderUtil.cc

```diff
@@ -2230,6 +2364,104 @@ void RenderUtilPrivate::LowlightNode(const rendering::NodePtr &_node)
   }
 }
 
+/////////////////////////////////////////////////
+void RenderUtil::ViewInertia(const Entity &_entity)
+{
+  std::vector<Entity> inrLinks;
+  if (this->dataPtr->modelToLinkEntities.find(_entity) !=
+           this->dataPtr->modelToLinkEntities.end())
+  {
+    std::vector<Entity> links = this->dataPtr->modelToLinkEntities[_entity];
+    for (const auto &link : links)
+      inrLinks.push_back(link);
+  }
+  else
+  {
+    inrLinks.push_back(_entity);
+  }
+
+  if (this->dataPtr->modelToModelEntities.find(_entity) !=
+      this->dataPtr->modelToModelEntities.end())
+  {
+    std::stack<Entity> modelStack;
+    modelStack.push(_entity);
+
+    std::vector<Entity> childModels;
+    while (!modelStack.empty())
+    {
+      Entity model = modelStack.top();
+      modelStack.pop();
+
+      inrLinks.insert(inrLinks.end(),
+          this->dataPtr->modelToLinkEntities[model].begin(),
+          this->dataPtr->modelToLinkEntities[model].end());
+
+      childModels = this->dataPtr->modelToModelEntities[model];
+      for (const auto &childModel : childModels)
+      {
+        modelStack.push(childModel);
+      }
+    }
+  }
+
+  // create and/or toggle inertia visuals
+
```

Comment:
```suggestion
```

Reply:
Fixed in [`9be5a99`](https://github.com/ignitionrobotics/ign-gazebo/pull/861/commits/9be5a9997478fe66a18f826ec83017d54c39fb22)